### PR TITLE
GP3 volumes support

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
## What does this PR change?
Added GP3 volumes support in AWS

## How was this PR tested?
Was tested in my setup, which uses GP3

## Does this PR require changes to documentation?
No

## Have you labeled this PR and its corresponding Issue as "next release" if it should be part of the next Opencost release? If not, why not?
Have no clue how to label

### CODE REVIEW

**Comment by AjayTripathy** [comment]
Thanks for this PR @AndrewChubatiuk . Is the most minimal set of changes to just modify aws.provider.go? I think the PR scope should be just limited to that-- we can discuss what's going on in queries.go in a separate PR or discussion.

File: pkg/prom/queries.go [hunk 1]
```diff
@@ -0,0 +1,483 @@
+package prom
```
Comment (AjayTripathy) [inline pkg/prom/queries.go]: Is this necessary within this PR? Happy to discuss changes here in a separate one!

**Review by AjayTripathy — CHANGES_REQUESTED** [review]

File: pkg/prom/queries.go (hunk 1 above)
Comment (AndrewChubatiuk) [reply pkg/prom/queries.go]: Sorry, unintended file here

Comment (AndrewChubatiuk) [reply pkg/prom/queries.go]: @AjayTripathy updated PR

**Review by AjayTripathy — APPROVED** [review]

File: pkg/prom/queries.go (hunk 1 above)
Comment (AjayTripathy) [reply pkg/prom/queries.go]: Thank you!

**Pull request merged**